DependencyInjection/DoctrineExtension.php

```diff
@@ -813,7 +813,9 @@ private function loadPropertyInfoExtractor($entityManagerName, ContainerBuilder
      */
     private function loadValidatorLoader(string $entityManagerName, ContainerBuilder $container) : void
     {
-        if (! interface_exists(LoaderInterface::class) || ! class_exists(DoctrineLoader::class)) {
+        $autoMapping = $container->hasParameter('validator.auto_mapping') ? $container->getParameter('validator.auto_mapping') : false;
```

Comment:
Hmm, I don't think this will work here - the `$container` in an extension class only has access to the parameters and services defined IN that class - not the whole container. This would need to be removed via a compiler pass, right?